tests/conftest.py

```diff
@@ -23,6 +22,7 @@
     TestServer,
     make_mocked_request,
 )
+from aiohttp.typedefs import JSONDecoder
```

Comment:
Previously the wrong `JSONDecoder` type hint was imported (from `json` instead of from `aiohttp.typedefs`)